Dockerfile.s390x

```diff
@@ -133,7 +133,7 @@ RUN useradd --create-home --gid docker unprivilegeduser
 
 VOLUME /var/lib/docker
 WORKDIR /go/src/github.com/docker/docker
-ENV DOCKER_BUILDTAGS apparmor selinux
+ENV DOCKER_BUILDTAGS apparmor pkcs11 selinux
```

Comment:
Ya I just want to make sure ping @estesp

On Tuesday, March 15, 2016, David Lawrence <notifications@github.com> wrote:

> In Dockerfile.s390x
> <https://github.com/docker/docker/pull/21003#discussion_r56207250>:
>
> > @@ -133,7 +133,7 @@ RUN useradd --create-home --gid docker unprivilegeduser
> >
> >  VOLUME /var/lib/docker
> >  WORKDIR /go/src/github.com/docker/docker
> > -ENV DOCKER_BUILDTAGS apparmor selinux
> > +ENV DOCKER_BUILDTAGS apparmor pkcs11 selinux
>
> Do we support the docker CLI on those systems? Bear in mind trust all
> happens on the CLI side.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/21003/files#r56207250>
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>